### PR TITLE
Testwide code coverage fix

### PR DESCRIPTION
**Pull request by philippbayer**
This could fix the code coverage test coverage badge, but who knows

### CODE REVIEW

**Comment by tsujigiri** [comment]
The Codeclimate token is already configured as an environment variable over at https://travis-ci.org/openSNP/snpr/settings. Things like this don't really belong anywhere where everyone can see them. Not that I expect anyone to send them false coverage reports. :wink: Still...

File: test/test_helper.rb [hunk 1]
```diff
@@ -2,7 +2,7 @@
 ENV['RAILS_ENV'] = 'test'
 unless ENV['CI']
   require 'simplecov'
-  SimpleCov.start('rails')
+  SimpleCov.start
```
Comment (tsujigiri) [inline test/test_helper.rb]: I'm not sure this is what we want. 😬

Comment (gedankenstuecke) [inline test/test_helper.rb]: meaning?

**Comment by gedankenstuecke** [comment]
> The Codeclimate token is already configured as an environment variable over at https://travis-ci.org/openSNP/snpr/settings. Things like this don't really belong anywhere where everyone can see them. Not that I expect anyone to send them false coverage reports

Ah, @philippbayer and I were following *code climate*'s own how-to and they recommend doing it like this. But of course you're right about it not being nice. I regenerated a new token, put this one in the *Travis CI* backend and deleted the old one from the `.travis.yml`, still seems to work now. :+1:

**Pull request merged**